src/main/java/seedu/simplykitchen/model/util/ComparatorUtil.java

```diff
@@ -24,4 +31,41 @@
                 food1.getDescription().fullDescription.compareTo(food2.getDescription().fullDescription);
     public static final Comparator<Food> SORT_BY_FIRST_CHARACTER = (food1, food2) ->
                 food1.getDescription().compareFirstCharacterTo(food2.getDescription());
+
+    /**
+     * Returns the foodInventorySortingComparators information according to {@code comparatorDescription}.
+     */
+    public static Comparator<Food>[] getComparator(String comparatorDescription) {
+        switch (comparatorDescription) {
+        case "default without ordering":
+            return DEFAULT_COMPARATOR_ARRAY;
+        case "ascending expiry date":
+            return SORT_EXPIRY_COMPARATORS;
+        case "descending priority":
+            return SORT_PRIORITY_COMPARATORS;
+        case "description":
+            return SORT_DESC_COMPARATORS;
+        default:
+            assert false : "Comparator is not defined.";
+            return null;
```

Comment:
Perhaps can throw an "IllegalArgumentException" instead of returning null?

Reply:
Will do, thanks!